lib/assets/stylesheets/modules/_search.scss

```diff
@@ -19,6 +12,48 @@
   }
 }
 
+.js .search__label {
+  @extend .govuk-visually-hidden;
+}
+
+.search__form {
+  position: relative;
+  width: 100%;
+}
+
+.search__input {
+  width: 230px;
+  width: calc(100% - #{$input-size});
+}
+
+.search__button {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  border: 0;
+  background-color: govuk-colour('blue');
+  color: #fff;
+  cursor: pointer;
+  border-radius: 0;
+  padding: 0;
+  width: $input-size;
+  height: $input-size;
```

Comment:
With the height as a fixed px size if you zoom in on the page in Firefox with 'Zoom text only' turned on, the button isn't tall enough next to the input.

<img width="291" alt="Screenshot 2021-03-08 at 13 10 19" src="https://user-images.githubusercontent.com/861310/110325911-dae15f80-800f-11eb-8d96-6c0548e588ff.png">

This can be fixed by changing the height to `100%`.

Reply:
Thanks for flagging this, I forgot to test this with text magnification. Have implemented the solution you suggested.